CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
-# 1.22.4 (2024-09-6)
+# 1.22.5 (2024-09-6)
 
 ## Updates/Fixes
 
-- add saved address to send screen [#343]()
+- add saved address to send screen [#343](https://github.com/asgardex/asgardex-desktop/pull/349)
+- fix asset mismatch and lp retrieval tool [350](https://github.com/asgardex/asgardex-desktop/pull/351)
```

Comment:
This link seems to be wrong... should link to:

    [#352](https://github.com/asgardex/asgardex-desktop/pull/352)